### PR TITLE
Assignment seven

### PR DESCRIPTION
**Pull request by kkorn2019**
Are we supposed to test the JAXB classes? I wasn't sure how to go about that. I think I have everything else done for now. 

I pushed some more changes to assignment 7 tonight. All tests are passing in CircleCi. I was able to get the sql query to work, with only one insert command at a time. If I put them all together into one String, it gives a syntax error. 

Also, I'm still not sure why the xml is coming back with the </stock> vs /> at the end but updating the comparison string makes the test pass. 

Of course, this is showing Assignment 6 files too, since that pull request hasn't been merged yet.

Thanks, 
Kayla

### CODE REVIEW

File: src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/apps/stockquote/BasicStockQuoteApplication.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+import java.util.List;
+
+/**
+ * A simple application that shows the StockService in action.
+ */
+public class BasicStockQuoteApplication {
+
+    private final StockService stockService;
+
+    // an example of how to use enum - not part of assignment 3 but useful for assignment 4
```
Comment (smarks) [inline src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/apps/stockquote/BasicStockQuoteApplication.java]: good practice to remove unneeded or stale comments.

File: src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/apps/stockquote/BasicStockQuoteApplication.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            StockQuery stockQuery = new StockQuery(args[0], args[1], args[2], interval);
+            StockService stockService = ServiceFactory.getStockServiceInstance();
+            BasicStockQuoteApplication basicStockQuoteApplication =
+                    new BasicStockQuoteApplication(stockService);
+            basicStockQuoteApplication.displayStockQuotes(stockQuery);
+
+        } catch (ParseException e) {
+            exitStatus = ProgramTerminationStatusEnum.ABNORMAL;
+            programTerminationMessage = "Invalid date data: " + e.getMessage();
+        } catch (StockServiceException e) {
```
Comment (smarks) [inline src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/apps/stockquote/BasicStockQuoteApplication.java:136]: good job here.

File: src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/model/StockQuote.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    private final String symbol;
+
+    /**
+     * Create a new instance of a StockQuote.
+     *
+     * @param price  the share price for the given date
+     * @param date   the date of the share price
+     * @param symbol the stock symbol.
+     */
+
```
Comment (smarks) [inline src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/model/StockQuote.java]: no blank line needed here.

**Review by smarks — APPROVED** [review]

File: src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/model/StockQuote.java (hunk 3 above)
Comment (kkorn2019) [reply src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/model/StockQuote.java]: removed

File: src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/apps/stockquote/BasicStockQuoteApplication.java (hunk 1 above)
Comment (kkorn2019) [reply src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/apps/stockquote/BasicStockQuoteApplication.java]: removed

File: src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/model/database/StockSymbol.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+ */
+@Entity
+@Table(name = "stock_symbol", schema = "", catalog = "stocks")
+public class StockSymbol implements DatabasesAccessObject {
+    private int id;
+    private String symbol;
+    
+    public StockSymbol(){
+        
+    }
```
Comment (smarks) [inline src/main/java/edu/KaylaKornelis/advancedjava/Assignment6/model/database/StockSymbol.java:21]: No javadoc here

**Pull request merged**